traffic_ops/app/db/migrations/2022100610190300_update_rascal_to_traffic_monitor.up.sql

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * 'License'); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+UPDATE TYPE
+SET name = 'TRAFFIC_MONITOR',
+    description = 'Traffic Monitor polling & reporting'
+WHERE name = 'RASCAL';
+
+UPDATE PARAMETER
+SET config_file = REPLACE(config_file, 'rascal', 'traffic_monitor')
+WHERE config_file = 'rascal-config.txt' OR config_file = 'rascal.properties';
+
+UPDATE PARAMETER
+SET value = 'TRAFFIC_MONITOR_TOP'
+WHERE value = 'RASCAL_TOP' AND name = 'latest_traffic_monitor';
```

Comment:
I don't see any instances of `RASCAL_TOP` in the repository - what is this supposed to do?

Reply:
Does that have some kind of use in a proprietary plugin or tool? Because from ATC's perspective, that parameter is meaningless unless it's assigned to a server that runs t3c (i.e. a cache server.

Reply:
I don't know but I can ask CP team.

Reply:
I believe this parameter is related to TMs created for distributed TM feature.

Reply:
Most likely will be removing this param from prod since there isn't a specific use case.